ksm/ksm_day6/예산.js

```diff
@@ -0,0 +1,16 @@
+function solution(d, budget) {
+    let total = 0;
+    let count = 0;
+    const sortedD = [...d].sort((a, b) => a - b); // 오름차순 정렬 
```

Comment:
```suggestion
    const sortedD = [...d].sort((a, b) => a - b); // 오름차순 정렬 
    let total = 0;
    let count = 0;
```
상수가 일반 변수보다 상단에 위치시키는게 일반적인 규칙입니다 🤗